test/geo/t_spatial.clj

```diff
@@ -254,3 +254,20 @@
                         (map dist)
                         (filter (partial > 50))
                         count) => (roughly 250 10))))))
+
+(facts "features"
+       (let [g1 (s/jts-point 0 0)
+             p1 {:name "null"}
+             f1 (s/->Feature g1 p1)
+             f2 (s/map->Feature {:geometry g1
+                                 :properties p1
+                                 :additional "information"})]
```

Comment:
Does it throw away `:additional` here?

Reply:
Additional fields are retained - see 86af4ad for a test to that end.